source/features/small-user-avatars.tsx

```diff
@@ -7,6 +7,14 @@ import observe from '../helpers/selector-observer.js';
 import getUserAvatarURL from '../github-helpers/get-user-avatar.js';
 import './small-user-avatars.css';
 
+function getAvatarUrlSafely(username: string, size: number): string {
+	try {
+		return getUserAvatarURL(username, size) || '';
```

Comment:
Why would this throw ? The username is just a string. Any string will product a URL without errors. If it throws, then it should be fixed.